ccd-definition/AuthorisationCaseEvent/AuthorisationCaseEvent.json

```diff
@@ -460,5 +460,19 @@
     "CaseEventID": "draftCMO",
     "UserRole": "caseworker-publiclaw-solicitor",
     "CRUD": "CRU"
+  },
+  {
+    "LiveFrom": "01/01/2017",
+    "CaseTypeID": "CARE_SUPERVISION_EPO",
+    "CaseEventID": "actionCMO",
+    "UserRole": "caseworker-publiclaw-judiciary",
```

Comment:
It looks not quite necessary since judges have `caseworker-publiclaw-courtadmin` as well.

Reply:
thanks for spotting @geek-soft